style.css

```diff
@@ -0,0 +1,254 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 1fr 2.5fr 1fr;
+  font-family: sans-serif;
```

Comment:
Always nice to have another font-family in your project 🙂

Reply:
very helpful, thank you!